.github/workflows/ci.yml

```diff
@@ -68,7 +69,40 @@ jobs:
               run: "forge build --sizes"
 
             - name: "Run the tests"
```

Comment:
I think we can rename this step to "Run proposal simulator test" and add one more step named "addresses tests" instead of adding a new job.

Reply:
I prefer to run them in parallel, adding them as a step would make them run sequentially. What do you think @prateek105 @prateek105 ?

Reply:
@ElliotFriedman I didn't ask about the name but about running in parallel or sequentially

Reply:
I think address tests won't take much time and can be run sequencially.